Lib/test/test_doctest/test_doctest.py

```diff
@@ -4,7 +4,7 @@
 
 from test import support
 from test.support import import_helper
-from test.support.pty_helper import FakeInput  # used in doctests
+from test.support.pty_helper import FakeInput  # noqa: F401
```

Comment:
Why `FakeInput` is needed?

Reply:
It's used in doctests.

Reply:
Could you add a comment for this?

Or, why not import it in doctests?

Reply:
> Or, why not import it in doctests?

I created https://github.com/python/cpython/pull/120679 to move imports inside doctests.